src/java.base/unix/classes/sun/nio/fs/UnixPath.java

```diff
@@ -905,12 +905,20 @@ public Path toRealPath(LinkOption... options) throws IOException {
             }
             final UnixFileKey elementKey = attrs.fileKey();
 
+            // Obtain the directory stream pointer. It will be closed by
+            // UnixDirectoryStream::closeImpl.
```

Comment:
I think you can replace "closeImpl" with "close" in the comment, that will be a bit clearer as the try-with-resources is just below it.

Reply:
Good point. So changed in ac0936c78b24ebe5132f89018dd905ea043ce92b.